GlotPress/GlotPress-WP
# 1454: Add left margin to warning discard link

## What?
Add some spacing between the warnings messages and its Discard link.

## Why?
On large warning messages, the Discard link gets side by side with the warning, with no margin.
Fixes #1454 

## How?
Add a 0.5em left margin to the Discard link on the warnings.

## Testing Instructions
1. Use last develop version with added warnings for start and ending spaces.
2. Add a string with both start and ending spaces to produce an warning for both spaces being added.
3. See the warning side by side with the Discard link with no spacing.

## Screenshots
Before:
![imagem](https://user-images.githubusercontent.com/7371591/173803162-a6dfc438-fdd4-4c56-b5f0-77517b2b5074.png)

After this PR:
![imagem](https://user-images.githubusercontent.com/7371591/173805017-ae55a626-3fbf-43f4-b6a2-2117e95beb98.png)
